test/integration/cuda/test_cuda_analytical_rosenbrock.cpp

```diff
@@ -3,160 +3,212 @@
 
 #include "../analytical_policy.hpp"
 #include "../analytical_surface_rxn_policy.hpp"
+#include "../oregonator.hpp"
+#include "../e5.hpp"
+#include "../hires.hpp"
```

Comment:
All these three tests fail on Derecho's A100 GPU somehow.

Reply:
After discussion with Kyle, we do not want to include these three tests into the CUDA integration test suite because they hardcode the Jacobian and forcing terms, and we need to write specific CUDA kernels for the GPU tests.